apiary.apib

```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
 
 # Group aiakos v1
 
+## Regions [/v1/support]
+
+Get the region list with keys
+
+### Get list [GET]
+
```

Comment:
if you need to return a json with more metadata (something like you propouse), I should add a function that iterate over the json, in order to generate a new json with your approach.